TKMovesets2/GameSpecific/Structures/Structs_t8.h

```diff
@@ -794,14 +855,66 @@ namespace StructsT8_gameAddr //todo
 	{
 		gameAddr requirements_addr;
 		uint32_t extraprop; // 881 list end value & extraprop values
-		uint32_t value; // Often small values
+		uint32_t value1;
+		uint32_t value2;
+		uint32_t value3;
+		uint32_t value4;
+		uint32_t value5;
+	};
+
+	struct ExtraMoveProperty //todo
+	{
+		uint32_t starting_frame;
+		uint32_t _0x4_int;
+		gameAddr* requirements_addr;
+		uint32_t id;
+		union
+		{
+			struct
+			{
+				uint32_t value1_unsigned;
+				uint32_t value2_unsigned;
+				uint32_t value3_unsigned;
+				uint32_t value4_unsigned;
+				uint32_t value5_unsigned;
+			};
+			struct
+			{
+				int32_t value1_signed;
+				int32_t value2_signed;
+				int32_t value3_signed;
+				int32_t value4_signed;
+				int32_t value5_signed;
+			};
+			struct
+			{
+				float value1_float;
+				float value2_float;
+				float value3_float;
+				float value4_float;
+				float value5_float;
+			};
+
+		};
+	};
+
+
+	struct UnknownNew
+	{
+		uint32_t _0x0;
+		uint16_t _0x4;
+		uint16_t _0x6;
+		gameAddr* requirements_addr;
+		uint32_t _0xC;
+		uint32_t _0x10;
 	};
 
 	struct Move
 	{
+		char encyption_key[8]; // not 100% confirmed yet
 		gameAddr name_addr;
 		gameAddr anim_name_addr;
-		gameAddr anim_addr;
+		char anim_addr[8]; // now encrypted
```

Comment:
You can use the `Byte` type here, it'll work even if this specific file doesn't define it